ocs_ci/ocs/cluster.py

```diff
@@ -1551,6 +1552,25 @@ def is_flexible_scaling_enabled():
     return failure_domain == "host" and flexible_scaling
 
 
+def check_ceph_health_after_add_capacity():
+    ceph_health_tries = 80
+    ceph_rebalance_timeout = 1800
+    if config.RUN.get("io_in_bg") and config.RUN.get("io_load"):
```

Comment:
I don't think we need to examine both `io_in_bg` and `io_load`.
`io_in_bg` is enough.

Maybe for `io_load` it would be nice to increase the timeout by the amount of load but this is not a must for now

Reply:
Yes, it is possible. I thought it would be more clear if we check both params. The increased timeout is compared to the results saw in the failed tests.

Reply:
Fixed